README.md

```diff
@@ -130,7 +134,19 @@ Where:
 
 ## Profile Retrieval
 
-When your application receives a token via the exposed endpoint (it will be assigned to a query string parameter named `token`), you can easily retrieve the user profile by adding the following to your endpoint handler:
+### Callback URl
+
+To allow sharing a Users profile with your application, you must expose a GET endpoint over https that Yoti will use to push a one-time token to your back end.
```

Comment:
We've gone with `one time use token` elsewhere, so would be good to have that here too instead of `one-time token` for consistency. Also can we capitalise `HTTPS`?

Reply:
Ok.